permission-portal-server/routes/organization.js

```diff
@@ -11,15 +12,16 @@ organizationRouter.use(bodyParser.json());
 
 const getOrgs = async (req, res, next) => {
   try {
-    const results = await pool.query("SELECT * FROM Organizations");
+    const client = await pool.connect();
+    const results = await client.query('SELECT * FROM "Organization"');
```

Comment:
Do folks have strong opinions on ORMs? I'm generally a fan of ones that have enough power to let you get under the hood when you need to. I'm not familiar enough with Sequelize - https://sequelize.org - but it does have a migration framework, which seems like something we should consider.